vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -17,7 +17,7 @@ def save_vms_inventory(ems, hashes, target = nil)
     log_header = "MIQ(#{self.name}.save_vms_inventory) EMS: [#{ems.name}], id: [#{ems.id}]"
 
     disconnects = if target.kind_of?(ExtManagementSystem) || target.kind_of?(Host)
-      target.vms_and_templates(true).dup
+      target.vms_and_templates(true).to_a.dup
```

Comment:
We are going to need this ALL over the refresh code

@kbrock Can you help out @tenderlove here, since you're already in this code?